br/backup-and-restore-faq.md

```diff
@@ -22,9 +22,7 @@ To reduce the impact on the cluster, you can use the `--ratelimit` parameter to
 
 ## Does BR back up system tables? During data restoration, do they raise conflict?
 
-The system schemas (`information_schema`, `performance_schema`, `mysql`) are filtered out during full backup. For more details, refer to the [Backup Principle](/br/backup-and-restore-tool.md#implementation-principles).
-
-Because these system schemas do not exist in the backup files, no conflict occurs among system tables during data restoration.
+Before v5.1.0, BR filtered out data from the system schema mysql during the backup. Since v5.1.0, BR backs up all data by default, including the system schemas mysql.*. But the technical implementation of restoring the system tables in mysql.* is not complete yet, so the tables in the system schema mysql are not restored by default. For more details, refer to the [Back up and restore table data in the mysql system schema](/br/backup-and-restore-tool.md#back-up-and-restore-table-data-in-the-mysql-system-schema-experimental-feature).
```

Comment:
```suggestion
Before v5.1.0, BR filtered out data from the system schema `mysql` during the backup. Since v5.1.0, BR **backs up** all data by default, including the system schemas `mysql.*`. But the technical implementation of restoring the system tables in `mysql.*` is not complete yet, so the tables in the system schema `mysql` are **not** restored by default. For more details, refer to the [Back up and restore table data in the `mysql` system schema (experimental feature)](/br/backup-and-restore-tool.md#back-up-and-restore-table-data-in-the-mysql-system-schema-experimental-feature).
```